app/src/main/java/com/mapbox/dash/example/MainApplication.kt

```diff
@@ -122,6 +124,30 @@ class MainApplication : Application() {
 
             customValues[CustomKeys.ENABLE_QUICK_SEARCH_SUGGESTIONS_IN_ACTIVE_GUIDANCE] = false
         }
+
+        configureEvProvider()
+    }
+
+    /**
+     * Configure [EvDataProvider] for Electric Vehicle Data
+     *
+     * Reference Docs https://docs.mapbox.com/android/navigation/ux/guides/coordination/electric-vehicle/
+     * Interface Docs https://docs.mapbox.com/android/navigation/api/uxframework/1.0.0-beta.40/ev-api/com.mapbox.dash.ev.api/-ev-data-provider/?query=interface%20EvDataProvider
+     */
+    private fun configureEvProvider() {
+        val evDataProvider = object : EvDataProvider {
```

Comment:
Btw, it will not be possible to change debug settings of this provider. Are we OK with this?

Reply:
I think it shouldn't be a problem for the sample app.